Documentation/connectors/ldap.md

```diff
@@ -126,9 +126,8 @@ connectors:
       # Following list contains field pairs that are used to match a user to a group. It adds an additional
       # requirement to the filter that an attribute in the group must match the user's
       # attribute value.
-      userMatchers:
-      - userAttr: uid
-        groupAttr: member
+      userAttr: uid
```

Comment:
I haven't checked the upstream issue here, but `userMatchers` are the way to go, please check: https://github.com/dexidp/dex/blob/30cd592801d493b9f253d7bf1c12b072b917d6a6/connector/ldap/ldap.go#L137

Reply:
I checked and it is not working with userMatchers ... So maybe in fact the issue is in the chart

Reply:
well, in dex's master those are deprecated but look at dex v2.22.0, the one argocd uses: https://github.com/dexidp/dex/blob/26061f9558f3011bea53ef5a5bb02ac669d612e0/connector/ldap/ldap.go#L135

so it's either 'downgrade' docs to match dex config or upgrade dex i guess :)